examples/merkle-tree-cli/Cargo.toml

```diff
@@ -11,7 +11,7 @@ path = "src/main.rs"
 
 [dependencies]
 clap = { version = "4.4.6", features = ["derive"] }
-lambdaworks-crypto = { workspace = true }
+lambdaworks-crypto = { workspace = true, features = ["serde"]}
```

Comment:
This came up when building all other examples. The crate relies on serde and it fails to compile if it's not added.